packages/testcafe-models/dataGrid/index.ts

```diff
@@ -217,13 +219,22 @@ export default class DataGrid extends Widget {
     return this.element.find(`.${this.addWidgetPrefix(CLASS.filterRow)}`);
   }
 
+  getFilterRowSearchIcon(columnIndex?): Selector {
+    return this.getFilterCell(columnIndex ? columnIndex : 0).find(`.${CLASS.filterMenuItem}`);
+  }
+
   getFilterCell(columnIndex: number): Selector {
     return this.getFilterRow().find(`[aria-colindex='${columnIndex + 1}']`);
   }
 
   getFilterRangeOverlay(): Selector {
     return this.body.find(`.${this.addWidgetPrefix(CLASS.filterRangeOverlay)}`);
   }
+
+  getFilterMenuOverlay(): Selector {
```

Comment:
Please use [the following approach](https://github.com/DevExpress/DevExtreme/blob/d37b38b6bfbdf8fa28f9d31278beaeea9d6090a1/e2e/testcafe-devextreme/tests/dataGrid/filterRow/filterRow.ts#L40) to work with filter menu

Reply:
Hi @Raushen , how do you mean by this? I see the link points to FilterEditor. Also to add, Filter Menu is found in an overlay wrapper which is outside the datagrid element..